### PR TITLE
pgbouncer: add exec to make sure signals are handled correctly

### PR DESCRIPTION
**Pull request by igorwwwwwwwwwwwwwwwwwwww**

### CODE REVIEW

**Comment by igorwwwwwwwwwwwwwwwwwwww** [comment]
The db setup is failing due to `travis-migrations/master/db/main/structure.sql` being re-created on a newer postgres version. Going to try running the build on trusty.

File: bin/pg_bouncer [hunk 1]
```diff
@@ -4,8 +4,12 @@ cd "$(dirname "$0")/.."
 
 if [ ! -f "bin/start-pgbouncer-stunnel" ]; then
   echo "warning: pgbouncer buildpack not found, setting PGBOUNCER_ENABLED=false"
-  exit
+  export PGBOUNCER_ENABLED=false
 fi
 
-export PGBOUNCER_PREPARED_STATEMENTS=false
-bin/start-pgbouncer-stunnel "$@"
+if [ "$PGBOUNCER_ENABLED" = '1' ] || [ "$PGBOUNCER_ENABLED" = 'true' ]; then
```
Comment (svenfuchs) [inline bin/pg_bouncer]: I kinda decided I didn't want this extra opt-in in this application. If a process is prefixed with `pg_bouncer` then it should be used. :)

Comment (igorwwwwwwwwwwwwwwwwwwww) [reply bin/pg_bouncer]: updated it to explicitly error out when the buildpack is missing

**Comment by igorwwwwwwwwwwwwwwwwwwww** [comment]
this was addressed in 918e842568f57349de08ed97c7a9cbd2d227432d, closing.

**Pull request closed without merging**